samples/display-gaming-interstitial-ad/sample.ts

```diff
@@ -41,47 +46,62 @@ function defineGamingInterstitialSlot() {
   // Slot returns null if the page or device does not support interstitials.
   if (gamingInterstitialSlot) {
     gamingInterstitialSlot.addService(googletag.pubads());
+    if (!delay) {
+      googletag.display(gamingInterstitialSlot);
```

Comment:
Why not move this after the call to `defineGamingInterstitialSlot()` in the `gameManualInterstitialSlotClosed` callback? Then we wouldn't need to have this delay parameter.

Reply:
I put the `display` inside this function because it sounds like a call to "show" the interstitial rather than "load" it, and including it here might make it less confusing. Also, if we move it outside, we need to check if `gamingInterstitialSlot` is not null.

But I don't have a preference, please see the last commit of a outside version :)